addons/l10n_nl/migrations/12.0.3.0/post-migration.py

```diff
@@ -1,12 +0,0 @@
-# Copyright 2019 Onestein (<https://www.onestein.eu>)
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-from openupgradelib import openupgrade
-
-
-@openupgrade.migrate()
-def migrate(env, version):
-    openupgrade.delete_records_safely_by_xml_id(
```

Comment:
Then, instead of this, you have to remove the corresponding ir.model.data, to avoid the tag being deleted by odoo later.

Reply:
Better to mark it as noupdate=1 for having traces.

Reply:
Ah yes, that makes sense. May I ask what problem leaving the xmlid in ir.model.data might cause?

Reply:
If it is noupdate=0, then odoo will remove that ir.model.data, and by doing that, will remove the tag. If the tag don't have ir.model.data or the ir.model.data has noupdate=1, then the tag will not be removed.

Reply:
I think also it is better to leave the xmlid for having traces. Maybe in a migration script later versions will need to make us of that xmlid.